docs/docsite/rst/dev_guide/developing_modules_in_bulk.rst

```diff
@@ -0,0 +1,160 @@
+* `lib/ansible/modules/$area/$prefix/$yourfirstmodule.py` - A single module. *Required new file*
+* `lib/ansible/utils/module_docs_fragments/$prefix.py` - Code documentation, such as details regarding common arguments. *Optional new file*
+* `lib/ansible/module_utils/$prefix.py` - Code shared between more than one module, such as common arguments. *Optional new file*
+*  `docs/docsite/rst/dev_guide/developing_module_utilities.rst` - Document your new `module_utils` file. *Optional update to existing file*
+
+And that's it.
+
+Before pushing your PR to GitHub it's a good idea to review the :doc:`developing_modules_checklist` again
+
+After publishing your PR on https://github.com/ansible/ansible a Shippable CI test should run (generally within a few minutes), check the results (at the end of the PR page) and ensure it's passing (green), if not inspect each of the results. Most of the errors should be self explanatory and are generally related to badly formatted documentation (see :doc:`YAMLSyntax`) or code that isn't valid Python 2.4 & Python 2.6 (see :doc:`developing_modules_python3`). If you aren't sure what a Shippable test message means copy it into the PR and add as a comment and we will review.
+
+If you need further advice join the `#ansible-devel` IRC channel (details in "Where to get support")
+
+
+Please don't force push your PRs, we prefer individual commits in the same PR, the PR will be squashed and merged by the Ansible Core Team when it's ready to be accepted. This is required to allow reviews to see what's changed since they started looking at your PR.
+
+We have a "ansibot" helper that comments on GitHub Issues and PRs which should highlight important information.
+
+
+Subsequent PRs
+``````````````
+
+By this point you first PR that defined the module namespace should have been merged. You can take the lessons learned from the first PR and apply it to the rest of the modules.
+
+Raise exactly one PR per module for the remaining modules.
+
+Over the years we've experimented with different sized PRs, some containing one module, some containing five, some even containing many tens of modules in, we've found the following:
+
+* A PR with a single file gets a higher quality review
+* PRs with multiple modules are harder for the creator to ensure all feedback has been applied
+* Lower priority to review. People generally review the easier things first, what would you review first a PR with one file, or five?
+
+FIXME, should we tell people to only raise one at a time, or (say) 5 PRs at once?
+
+As with your first PR, please don't "force push" updates to GitHub as it makes the reviewers (and submitters) life a lot harder.
+
+After each PR has been merged please update the `CHANGELOG.md <https://github.com/ansible/ansible/blob/devel/CHANGELOG.md>`_ so your module is listed under the "New Modules" section for the new release.
+
```

Comment:
I don't think we need to require them to do this, normally the one merging can easily add it or we catch them when doing release, which we have to do either way.